test/externalscene_test.js

```diff
@@ -0,0 +1,14 @@
+var itownsTesting = require('./itowns-testing.js');
+var example = require('../examples/externalscene.js');
+
+describe('External Scene', function() {
+    it('should use the user constructed scene', function(done) {
+        example.globeView.mainLoop.addEventListener('command-queue-empty', () => {
```

Comment:
You could use this event?
`example.globeView.addEventListener(itowns.GLOBE_VIEW_EVENTS.GLOBE_INITIALIZED`...